History.md

```diff
@@ -2364,7 +2364,7 @@
 3.2.0 / 2013-04-15
 ==================
 
-  * add "view" constructor setting to override view behaviour
+  * add "view" constructor setting to override view behavior
```

Comment:
This is just the British spelling of the word, which is fine. I would leave it as is to match the commit